webpack/scenes/ContentViews/Create/CreateContentViewModal.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Modal, ModalVariant } from '@patternfly/react-core';
```

Comment:
I may be biased, but consider using the `ForemanModal` component to be consistent with other modals.  Usage details here: http://foreman-storybook.surge.sh/?path=/story/components-foremanmodal-foremanmodal-usage--foreman-modal-basics

It hasn't been upgraded yet to use the PF4 modal component, but hopefully that shouldn't make too much visual difference..  (If so then it's fine to leave as is.)

Reply:
I think @johnpmitsch wanted to be on PF4 for the modal and form on this page instead of using Foreman Form and modal.